lib/image-inspector-client.rb

```diff
@@ -74,13 +75,10 @@ def http_options
     end
 
     def http_headers
-      if @auth_options[:bearer_token]
-        {
-          Authorization: "Bearer #{@auth_options[:bearer_token]}"
-        }
-      else
-        {}
-      end
+      headers = {}
+      headers[:Authorization] = "Bearer #{@auth_options[:bearer_token]}" if @auth_options[:bearer_token]
+      headers[:'X-Auth-Token'] = @auth_options[:auth_token] if @auth_options[:auth_token]
```

Comment:
the purpose of this change is to enable authentication of requests directly to the Image Inspector. The relevant changes to image inspector can be found here: https://github.com/moolitayer/image-inspector-client/pull/17

We use a custom auth header because the standard `Authorization` header is used & stripped by the kubernetes apiserver proxy.